extensions/data-workspace/src/common/workspaceTreeDataProvider.ts

```diff
@@ -115,6 +115,14 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
 	}
 
 	handleDrag(treeItems: readonly WorkspaceTreeItem[], dataTransfer: vscode.DataTransfer): void | Thenable<void> {
+		// don't do anything if trying to drag the project node since it isn't supported and there's an error when trying to
+		// convert the project node to JSON https://github.com/microsoft/azuredatastudio/issues/22529
+		const relativePath = treeItems[0].element?.relativeProjectUri?.fsPath.substring(1); // remove leading slash
```

Comment:
Why always the first tree item?  what does the `treeItems` list contain?  Can you kindly add some comments here to save me the trouble of looking up the drag/drop API documentation?

Reply:
updated comment with more info!